### PR TITLE
Remove error message for letter that is too long

### PR DESCRIPTION
**Pull request by CrystalPea**
This error will be instead defined in the notifications-admin app

We did this change so the CSS for this banner communicates with
styling for other elements on the page.

### CODE REVIEW

**Comment by quis** [comment]
I think you can remove these lines as well:
https://github.com/alphagov/notifications-utils/blob/f73fbda03467e558f53224f2b5f901e12d19c858/notifications_utils/template.py#L606-L608

File: tests/test_pdf.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ def test_pdf_page_count_src_pdf_not_a_pdf():
         pdf_page_count(BytesIO(file_data))
 
 
+@pytest.mark.parametrize("page_count, expected_result", [
+    (None, False), (3, False), (11, True)
```
Comment (quis) [inline tests/test_pdf.py]: Always best to test the boundary cases to make sure you haven’t made an off-by-one error:
```suggestion
    (None, False), (10, False), (11, True)
```

Comment (CrystalPea) [reply tests/test_pdf.py]: Good idea, done ☺️
